certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5073 @@
+        "parameters": [
+          {
+            "name": "operatingStatesId",
+            "in": "path",
+            "x-ms-url-encoding": "single",
+            "required": true,
+            "description": "The Id of the operatingState",
+            "x-ms-summary": "The Id of the operatingState",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MasterDataOperatingState"
+            }
+          },
+          "401": {
+            "description": "API Key not valid",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/miscellaneous/materials": {
+      "get": {
+        "description": "Get materials ",
+        "summary": "Get materials",
+        "tags": [
+          "miscellaneous"
+        ],
+        "operationId": "GetMaterials",
+        "x-ms-visibility": "internal",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MaterialsListFilter"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/operations/progress/workplaceId/{workplaceId}": {
+      "get": {
+        "description": "Get operations that are in progress",
+        "summary": "Get operations in progress",
+        "tags": [
+          "operations"
+        ],
+        "operationId": "OperationsInProgress",
+        "parameters": [
+          {
+            "name": "workplaceId",
+            "in": "path",
+            "x-ms-url-encoding": "single",
+            "required": true,
+            "description": "Id of workplace",
+            "x-ms-summary": "WorkplaceId",
+            "x-ms-dynamic-values": {
+              "operationId": "getWorkplaces",
+              "value-path": "id",
+              "value-title": "name",
+              "value-collection": "workplaces"
+            },
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/operations/dispatched/workplaceId/{workplaceId}": {
+      "get": {
+        "description": "Get operations that are dispatched",
+        "summary": "Get dispatched operations ",
```

Comment:
@dominikamann Please remove extra space at the end of the string

Reply:
@sriyen-msft Resolved, thank you for the information.